### PR TITLE
fix: change ci param to errorbar in sns-catplot

### PR DESCRIPTION
**Pull request by Busato**
#### What does this implement/fix?
Two tests were failing because of a deprecation of the `ci` parameter in `sns.catplot` [function](https://seaborn.pydata.org/generated/seaborn.catplot.html).

image

They were changed to `errorbar` instead that do the same thing when using None (both of the were using it)

Contributed with ❤️ by AE Studio

### CODE REVIEW

**Comment by larsoner** [comment]
CI failures appear unrelated, let me fix those in a separate PR then merge `main` into this branch, then should be good to go. Thanks in advance @Busato !

File: doc/changelog.rst [hunk 1]
```diff
@@ -197,3 +197,4 @@ Enhancements
 .. _Samuel Powell: https://github.com/samuelpowell
 .. _Johann Benerradi: https://github.com/HanBnrd
 .. _Florin Pop: https://github.com/florin-pop
+.. _Nicolas Busato: https://github.com/Busato
```
Comment (larsoner) [inline doc/changelog.rst:200]: Can you also add a line under the `v0.6.dev0` heading toward the top?

**Comment by larsoner** [comment]
You can now also remove this line that I added https://github.com/mne-tools/mne-nirs/pull/525/files#diff-15c86dcddb07192eeb2d84db2a5dfca48e3837471adbbd9b3e05425b44fce485R49 if you want to make sure it has been fixed everywhere

File: doc/changelog.rst (hunk 1 above)
Comment (Busato) [reply doc/changelog.rst:200]: @larsoner do you mean a line saying what the PR changed?

Comment (larsoner) [reply doc/changelog.rst:200]: Yep!

**Comment by rob-luke** [comment]
Thanks @Busato 🎉

**Pull request merged**